Mani-Quotify/app/src/main/java/com/mani/quotify007/view/utils/BottomNavItem.kt

```diff
@@ -0,0 +1,13 @@
+package com.mani.quotify007.view.utils
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Favorite
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.ui.graphics.vector.ImageVector
+
+sealed class BottomNavItem(var title: String, var icon: ImageVector, var route: String) {
+    data object Home : BottomNavItem("Home", Icons.Default.Home, "home")
```

Comment:
data class vs data object?

sort of interview question

Reply:
data class - if we need multiple instances with different data.
data object -if we need a single instance with fixed data.